### ISSUE TITLE
Exception trying to create customer address.

### ISSUE THREAD

**Issue by agostinofiscale** [signifly/laravel-shopify#42]
Hi everyone.

I'm developing a complex system that needs to syncronize users between Shopify and a proprietary CRM. What I found is that when i try to create a customer address using the following:

```
$shopify->createCustomerAddress(
    $shopifyCustomer->id,
    [
        "name" => $this->payload["Name"],
        "company" => $this->payload["Company"],
       ...
    ]
);
```

I got the following exception:

```
Undefined array key "address" {"exception":"[object] (ErrorException(code: 0): Undefined array key \"address\" at /Applications/MAMP/htdocs/craw/vendor/laravel/framework/src/Illuminate/Http/Client/Response.php:349)
```

and this happens because the code is looking for "address" meanwhile Shopify returns "customer_address" (https://shopify.dev/docs/api/admin-rest/2022-04/resources/customer-address#post-customers-customer-id-addresses).

I didn't give it a more in-depth look but maybe you already handle this situation in another case.

**Comment by egyjs** [comment]
same with me

i fixed in my local like this:
```php
    public function createCustomerAddress($shopify,$customerId, $data)
    {
        $resource = 'addresses';
        $key = Str::singular($resource);
        $resourceClass = $this->resourceClassFor($resource);
        $response = $shopify->post(implode('/', [...['customers',$customerId], "{$resource}.json"]), [$key => $data]);

        return new $resourceClass($response['customer_address'], $shopify);
    }

    protected function resourceClassFor(string $resource): string
    {
        $resourceClass = Str::of($resource)
            ->studly()
            ->singular()
            ->prepend('Signifly\\Shopify\\REST\\Resources\\')
            ->append('Resource');

        return class_exists($resourceClass) ? $resourceClass : ApiResource::class;
    }
```

## Note: this 2 functions is custom in my own class

**Comment by Razorsheep** [comment]
Fixed in v1.2.6
